docs/extend/modular-resources/create.md

```diff
@@ -0,0 +1,511 @@
+    Args:
+        address (str): The address to serve the module on
+    """
+    module = Module(address)
+    module.add_model_from_registry(Base.SUBTYPE, MyBase.MODEL)
+    await module.start()
+
+if __name__ == "__main__":
+    if len(sys.argv) < 2:
+        raise Exception("Need socket path as command line argument")
+
+    asyncio.run(main(sys.argv[1]))
+
+```
+
+{{% /expand%}}
+
+{{% alert title="Important" color="note" %}}
+
+You must define all functions belonging to a built-in resource type if defining a new model.
+Otherwise, the class won’t instantiate.
+
+- If you are using the Python SDK, raise an `NotImplementedError()` in the body of functions you do not want to implement or put `pass`.
+- If you are using the Go SDK, return `errUnimplemented`.
+- Additionally, return any values designated in the function's return signature, typed correctly.
+
+{{% /alert %}}
+
+### Compile the module into an executable
+
+To [add a module](/extend/modular-resources/configure/#configure-your-module) to the configuration of your robot, you need to have an [executable](https://en.wikipedia.org/wiki/Executable) that runs your module when executed, can take a local socket as a command line argument, and cleanly exits when sent a termination signal.
+
+Your options for completing this step are flexible, as this file does not need to be in a raw binary format.
+
+{{< tabs >}}
+{{% tab name="Go" %}}
+
+If using the Go SDK, use Go to [compile](https://pkg.go.dev/cmd/go#hdr-Compile_packages_and_dependencies) and obtain a single executable for your module:
+
+- Navigate to your module directory in your terminal.
+- Run `go build` to compile your entry point (main program) file <file>main.go</file> and all other <file>.go</file> files in the directory, building your module and all dependencies into a single executable file.
+- Run `ls` in your module directory to find the executable, which should have been named after the module directory.
+
+Expand the [Go module code](#code-a-main-program-that-serves-as-the-entry-point-file-to-the-module) to view <file>main.go</file> for an example of this.
+<file>main.go</file> adds the custom model <file>mybase.go</file> from the resource registry, while <file>mybase.go</file> defines and registers the module.
+
+{{% /tab %}}
+{{% tab name="Python" %}}
+
+If using the Python SDK, one option is to create and save a new shell script (<file>.sh</file>) that runs your module at your entry point (main program) file.
+
+For example:
+
+``` shell
+#!/bin/sh
+cd `dirname $0`
+
+# Be sure to use `exec` so that termination signals reach the python process,
+# or handle forwarding termination signals manually
+exec python3 -m <your-module-directory-name>.<main-program-filename-without-extension> $@
+```
+
+To make this shell script executable, run the following command in your terminal:
+
+``` shell
+sudo chmod +x <FILEPATH>/<FILENAME>
+```
+
+{{% /tab %}}
+{{% /tabs %}}
```

Comment:
```suggestion
{{< /tabs >}}
```
[nit] Just because the opening one is like this, so makes sense to match them I think